src/task/script.js

```diff
@@ -291,15 +293,22 @@ const handleDateChange = (event) => {
 function handleStatusChange(event = { target: { value: 'AVAILABLE' } }) {
   const assignee = document.getElementById('assigneeInput');
   const endsOnWrapper = document.getElementById('endsOnWrapper');
+  const featureRadio = document.getElementById('feature');
+
   if (event.target.value === 'ASSIGNED') {
     setDefaultDates();
-    assignee.style.display = '';
+    assignee.classList.add('show-assignee-field');
+    assignee.style.display = 'none';
     endsOnWrapper.style.display = '';
```

Comment:
Shouldn't we have a defined value instead of empty string?

https://developer.mozilla.org/en-US/docs/Web/CSS/display